exercises/rail-fence-cipher/src/RailFenceCipher.hs

```diff
@@ -0,0 +1,7 @@
+module RailFenceCipher (encode, decode) where
+
+encode :: Int -> [a] -> [a]
+encode = "You need to implement this function!"
+
+decode :: Int -> [a] -> [a]
```

Comment:
> I see you ask whether to change this. I would use `String` here simply because that is all tests will use, and it's fine if students realise it can be generalised and therefore change it back to `[a]`. I would therefore also not change the example, minus the comment I make on it.

same comment

Reply:
Ahh ok, dint read this before